### PR TITLE
add autoinstall for ubuntu 21.04

### PR DESCRIPTION
**Pull request by ZouYuhua**
add autoinstall for ubuntu 21.04

test items:
1) check autoinstall for ubuntu 20.10 live server and ubuntu 21.04 live server
2) check the default auto install

test result:
1) set cloud-init as autoinstall method for ubuntu 21.04:
    
image

2) set default autoinstall method for ubuntu 21.04:
image

### CODE REVIEW

File: linux/deploy_vm/ubuntu/reconfigure_ubuntu_iso_vm.yml [hunk 1]
```diff
@@ -18,6 +18,12 @@
     dest: ubuntu_install_method.yml
     mode: '0644'
 
+- name: Read ubuntu install method file
+  command: cat ubuntu_install_method.yml
+  register: file_result_ubuntu_install_method
+
+- debug: var=file_result_ubuntu_install_method
```
Comment (Tomorrow9) [inline linux/deploy_vm/ubuntu/reconfigure_ubuntu_iso_vm.yml:25]: debug also need a 'name'.

Comment (ZouYuhua) [reply linux/deploy_vm/ubuntu/reconfigure_ubuntu_iso_vm.yml]: Done

Comment (ZouYuhua) [reply linux/deploy_vm/ubuntu/reconfigure_ubuntu_iso_vm.yml]: @Tomorrow9 add the screenshots of test result

**Review by Tomorrow9 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**